CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.8]
```

Comment:
Please remove the changes made to this file. pyTenable - version 1.4.8 is already out. Instead, you could add these information to the PR description.

Reply:
Will do, thanks